src/test/java/org/fundacionjala/pivotal/hooks/Hooks.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.pivotal.hooks;
+
+import cucumber.api.java.After;
+import org.fundacionjala.pivotal.api.APIProjectsRequests;
+import static org.fundacionjala.pivotal.steps.api.APIProjectSteps.PROJECT_ENDPOINT;
+
+/**
+ * Hooks.java
+ * Class that will have hooks for scenarios.
+ */
+public class Hooks {
+    private APIProjectsRequests requests;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "requests" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)